.github/workflows/test_cli_action.yaml

```
@@ -0,0 +1,26 @@
+name: "CLI Action testing"
+
+on:
+  pull_request:
+    branches:
+      - "*"
+  push:
+    tags:
+      - "v*" # Push events to matching v*, i.e. v1.0, v20.15.10
+    branches:
+      - main
+
+env:
+  TERM: unknown  # Disables colors in rich
+
+jobs:
+  version:
+    name: "Check Snowflake CLI version"
```

The action is failing because config file is not defined in the repo, you can add a temporal config file with this command
```suggestion
    - name: "Create temporal config file"
       run: touch config.toml
    name: "Check Snowflake CLI version"
```